docs/client/chain.md

```diff
@@ -270,8 +271,8 @@ def process_block_header(state: State, block: Block) -> None:
 
 ```python
 def process_operations(state: State, body: BlockBody) -> None:
-    # process attestations
-    process_attestations(state, body.attestations)
+    attestations=[*body.attestations, body.proposer_attestation]
```

Comment:
These two items have differing types I think

body.attestations:

```py
attestations: List[ValidatorAttestation, VALIDATOR_REGISTRY_LIMIT]
```

```py
class ValidatorAttestation(Container):
    validator_id: uint64
    data: AttestationData
```

body.proposer_attestation:

```py
proposer_attestation: ProposerAttestationData
```

```py
class ProposerAttestationData(Container):
    target: Checkpoint
    source: Checkpoint
```

Reply:
this won't be changed at all, ProposerAttestationData will have no effect on this block's STF because its proposer vote for next block inclusion and for this slot will only be imported in forkchoice

(getting this changed)